crates/torii/core/src/constants.rs

```diff
@@ -1,3 +1,16 @@
 pub const TOKEN_BALANCE_TABLE: &str = "token_balances";
 pub const TOKEN_TRANSFER_TABLE: &str = "token_transfers";
 pub const TOKENS_TABLE: &str = "tokens";
+
+pub(crate) const LOG_TARGET: &str = "torii_core::engine";
+pub const QUERY_QUEUE_BATCH_SIZE: usize = 1000;
+
+pub const IPFS_URL: &str = "https://ipfs.io/ipfs/";
+pub const IPFS_CLIENT_MAX_RETRY: u8 = 3;
+
+pub const IPFS_CLIENT_URL: &str = "https://ipfs.infura.io:5001";
+pub const IPFS_CLIENT_USERNAME: &str = "2EBrzr7ZASQZKH32sl2xWauXPSA";
+pub const IPFS_CLIENT_PASSWORD: &str = "12290b883db9138a8ae3363b6739d220";
```

Comment:
In sozo we moved this to be read from env variables, and default to cartridge's if nothing is found. Maybe something we want here in a future, to enable usage of custom IPFS nodes.